index.js

```diff
@@ -87,8 +87,12 @@ function processLastItem(/* CODE HERE */) {
  * [2] Invoking `processSum` passing `[]` and `(num) => num + 1000`,
  * should return 1000.
 */
-function processSum(/* CODE HERE */) {
-  /* CODE HERE */
+function processSum(numberList, callback) {
+  return callback(numberList.reduce(sum, 0));
+}
+
+function sum(accum, item){
+  return accum + item;
 }
```

Comment:
I like the way you organized this problem.  Good job.  An alternate way would be to define the call back function as an arrow function directly in the reduce method.